paymentsheet/src/test/java/com/stripe/android/utils/screenshots/README.md

```diff
@@ -0,0 +1,38 @@
+# PaymentSheet screenshot tests
+
+We use [Paparazzi](https://github.com/cashapp/paparazzi) to run screenshot tests.
+
+## Creating screenshot tests
+
+To create a screenshot test, create a new test class and add the `PaparazziRule`. This rule takes any number of configuration enums and runs each test with all possible permutations.
+
+```kotlin
+class MyNewScreenshotTest {
+
+    @get:Rule
+    val paparazziRule = PaparazziRule(
+        SystemAppearance.values(),
+        PaymentSheetAppearance.values(),
+        // Add others…
+    )
+
+    @Test
+    fun testDefault() {
+        paparazziRule.snapshot {
+            // My UI goes here…
+        }
+    }
+}
+```
+
+## Recording screenshots
+
+To record screenshots, run `recordPaparazziDebug`. This will record all screenshots of the specified module. New or updated screenshots will be added to your Git staging area.
```

Comment:
Should we prefix the gradle commands with a `./gradlew `? Might help first time contributors.

Reply:
Will add!